docs/kintoblocks/jobs.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Job KintoBlock
+sidebar_label: Job
+---
+
+## What are Jobs?
+
+**Jobs** are KintoBlocks that support event-based tasks. A Job KintoBlock can be used to add various tasks to deployments for e.g. a scheduled job that dispatches notification emails. Jobs can be of two types - one-time or repeatable and the type of job can be specified in the Project integrated with the KintoBlock.
+
+## Creating a Job
+
+1. Select **KintoBlock** from the sidebar.
+
+2. Click on the **Create New KintoBlock** card.
+
+3. Select the **Job KintoBlock** flavor
+
+4. Fill the **Choose Repository** form by specifying the details pertaining to the linked repo.
+
+    ![Repository for Job](/docs/assets/choose-repository-job.png)
+
+5. Click on the **Continue** button.
+
+6. Next, specify the details on `Name`, `Internal Name` and `Source Type`.
+
+7. Click on the **Create Job** button to successfully create the KintoBlock.
+
+   ![Create Job](/docs/assets/create-job.png)
+
+## CRON Job or Configuration of Job
+
+Jobs created in KintoHub can be configured as `Run Once` or `Repeat Schedule`.
+
+The **Run Once** option can be configured in two ways:
+
+1. **Pre-Deployment**: This Job will be run before the project deployment is started.
+
+2. **Post-Deployment**: This Job will be run after the project deployed is completed.
+
+Pre-Deployment and Post-deployment jobs can be used in cases where a specific set of commands needs to be executed. For e.g. Running a package.json file for hosting a website.
```

Comment:
> For e.g. Running a package.json file for hosting a website.

Does not mean a lot I think.

"Running a database migration"

Reply:
Agreed - this sentence does not make sene.

Reply:
or sending a slack message.

Reply:
Done.